wazo_call_logd/plugins/cdr/api.yml

```diff
@@ -460,3 +487,44 @@ definitions:
         type: boolean
       filename:
         type: string
+  CDRStatistics:
+    properties:
+      items:
+        items:
+          $ref: '#/definitions/CDRStatistic'
+        type: array
+      total:
+        type: integer
+    type: object
+  CDRStatistic:
+    properties:
+      from:
+        description: Start of the statistic interval.
+        type: string
+      until:
+        description: End of the statistic interval.
+        type: string
+      tenant_uuid:
+        description: Tenant UUID of the corresponding call log.
+        format: uuid
+        type: string
+      received:
+        description: Total of calls received during the interval
+        type: integer
+      answered:
+        description: Total of answered calls during the interval
+        type: integer
+      not_answered:
+        description: Total of not answered calls during the interval
```

Comment:
We maybe want to give a more precise description to reach this condition
see: https://github.com/wazo-platform/wazo-call-logd/blob/master/wazo_call_logd/plugins/support_center/api.yml#L351
Same comment for other fields